WCFDataService/Client/System/Data/Services/Client/Binding/BindingUtils.cs

```diff
@@ -86,8 +86,9 @@ internal static void VerifyObserverNotPresent<T>(object oec, string sourceProper
 
             if (typedCollection.Observer != null)
             {
-                throw new InvalidOperationException(Strings.DataBinding_CollectionPropertySetterValueHasObserver(sourceProperty, sourceType));
```

Comment:
I'll add this: If ProxyBackingFieldNamingConvention is None we will throw as always.